code/game/turfs/closed/wall/mineral_walls.dm

```diff
@@ -55,6 +58,7 @@
 	icon_state = "bananium_wall-0"
 	base_icon_state = "bananium_wall"
 	sheet_type = /obj/item/stack/sheet/mineral/bananium
+	hardness = 95 //it's banana
```

Comment:
it's still a mineral, should still be hard.

Reply:
how hard....?

Reply:
That's up to you, not like it exists in real life, I just don't think it should be weaker than snow and bamboo...

Reply:
roger that captain